### PR TITLE
Some Spanish Translation Fixes

### PR DESCRIPTION
**Pull request by tadeu2**
Some Spanish Translation Fixes

### CODE REVIEW

File: localization/i18n/es/OrcaSlicer_es.po [hunk 1]
```diff
@@ -667,7 +667,7 @@ msgstr "Superficie"
 msgid "Horizontal text"
 msgstr "Texto horizontal"
 
-msgid "Shift + Mouse move up or down"
+msgid "Shift + Mouse move up or dowm"
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:670]: This should be down.

File: localization/i18n/es/OrcaSlicer_es.po [hunk 2]
```diff
@@ -914,7 +914,7 @@ msgid ""
 "Advanced options cannot be changed for the selected font.\n"
 "Select another font."
 msgstr ""
-"Las opciones avanzadas no pueden modificarse para la fuente seleccionada. "
+"Las opciones avanzadas no pueden modificarse para la fuente seleccionada. \n"
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:917]: No need to have a space before the line jump

File: localization/i18n/es/OrcaSlicer_es.po [hunk 3]
```diff
@@ -961,10 +961,10 @@ msgid "points"
 msgstr "puntos"
 
 msgid "Revert gap between characters"
-msgstr "Revert gap between characters"
+msgstr "Revertir el espacio entre caracteres"
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:964]: "Deshacer" migh be a better word here.

File: localization/i18n/es/OrcaSlicer_es.po [hunk 4]
```diff
@@ -961,10 +961,10 @@ msgid "points"
...
 
 msgid "Distance between characters"
-msgstr "Distancia entre caracteres"
+msgstr "Distancia entre carácteres"
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:967]: The oficial word from the RAE is caracteres and not carácteres. Please check: https://www.rae.es/desen/car%C3%A1cter

File: localization/i18n/es/OrcaSlicer_es.po [hunk 5]
```diff
@@ -1016,7 +1016,7 @@ msgstr "Orienta el texto hacia la cámara."
 
 #, boost-format
 msgid ""
-"Can't load exactly same font(\"%1%\"). Application selected a similar "
+"Can't load exactly same font(\"%1%\"). Aplication selected a similar "
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:1019]: This should stay as Application

File: localization/i18n/es/OrcaSlicer_es.po [hunk 6]
```diff
@@ -1676,7 +1676,7 @@ msgstr "Ancho de Extrusión"
 msgid "Wipe options"
 msgstr "Opciones de limpieza"
 
-msgid "Bed adhesion"
+msgid "Bed adhension"
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:1679]: This should stay as adhesion too.

File: localization/i18n/es/OrcaSlicer_es.po [hunk 7]
```diff
@@ -2232,16 +2238,16 @@ msgid_plural "Following model objects have been repaired"
 msgstr[0] "Se ha reparado el siguiente modelo de objeto"
 msgstr[1] "Se han reparado los siguientes objetos del modelo"
 
-msgid "Failed to repair following model object"
-msgid_plural "Failed to repair following model objects"
+msgid "Failed to repair folowing model object"
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:2241]: This should stay as following in both cases.

File: localization/i18n/es/OrcaSlicer_es.po [hunk 8]
```diff
@@ -2697,7 +2703,7 @@ msgstr "Tarea de envío de impresión fallida."
 msgid "Service Unavailable"
 msgstr "Servicio No Disponible"
 
-msgid "Unknown Error."
+msgid "Unkown Error."
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:2706]: Should stay as Unknown.

File: localization/i18n/es/OrcaSlicer_es.po [hunk 9]
```diff
@@ -2706,7 +2712,8 @@ msgstr "Enviando la configuración de impresión"
 #, c-format, boost-format
 msgid "Successfully sent. Will automatically jump to the device page in %ss"
 msgstr ""
-"Envío exitoso. Se saltará automaticamente a la página del dispositivo en %ss"
+"Envío exitoso. Se alternará automáticamente a la página del dispositivo en "
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:2715]: No need to add a space before the line jump

File: localization/i18n/es/OrcaSlicer_es.po [hunk 10]
```diff
@@ -2829,7 +2836,7 @@ msgid ""
 "contributors."
 msgstr ""
 "Slic3r fue creado por Alessandro Ranellucci con la ayuda de muchos otros "
-"contruyentes."
+"colaboradores."
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:2839]: La palabra correcta sería contribuyentes.

File: localization/i18n/es/OrcaSlicer_es.po [hunk 11]
```diff
@@ -2868,13 +2875,13 @@ msgstr "SN"
...
 msgstr ""
-"Ajustes de información de ranura AMS mientras la impresión no sea soportada"
+"Ajustes de información de ranura AMS mientras la impresión no tenga soportes"
 
 msgid "Factors of Flow Dynamics Calibration"
 msgstr "Factores de Calibración de Dinámicas de Caudal"
 
 msgid "PA Profile"
-msgstr "Perfil de Avance de Presión de Línea"
+msgstr "Perfil de Avance de Presión Líneal"
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:2884]: Its Lineal, not Líneal

File: localization/i18n/es/OrcaSlicer_es.po [hunk 12]
```diff
@@ -2930,7 +2937,7 @@ msgid "Nozzle temperature"
 msgstr "Temperatura de la boquilla"
 
 msgid "Bed Temperature"
-msgstr "Temperatura de Cama"
+msgstr "Temperatura de cama"
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:2940]: should stay as Cama since its Bed Temperature.

File: localization/i18n/es/OrcaSlicer_es.po [hunk 13]
```diff
@@ -3703,7 +3711,7 @@ msgstr ""
 "El valor se restablecerá a 0."
 
 msgid ""
-"Too large elephant foot compensation is unreasonable.\n"
+"Too large elefant foot compensation is unreasonable.\n"
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:3714]: It's elephant.

File: localization/i18n/es/OrcaSlicer_es.po [hunk 14]
```diff
@@ -4906,19 +4913,19 @@ msgid "Flow rate test - Pass 2"
...
 msgid "Retraction test"
-msgstr "Test de retracción"
+msgstr "Test de Retracciones"
 
 msgid "Orca Tolerance Test"
 msgstr "Test de Tolerancia Orca"
 
 msgid "Max flowrate"
-msgstr "Máximo caudal"
+msgstr "Test de Caudal Máximo"
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:4928]: Caudal Máximo, test word is not added in the original string.

File: localization/i18n/es/OrcaSlicer_es.po [hunk 15]
```diff
@@ -6243,8 +6247,8 @@ msgstr ""
 "El archivo %s ya existe\n"
 "¿Desea reemplazarlo?"
 
-msgid "Confirm Save As"
-msgstr "Salvar Como"
+msgid "Comfirm Save As"
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:6250]: Its Confirm.

File: localization/i18n/es/OrcaSlicer_es.po [hunk 16]
```diff
@@ -6468,7 +6472,7 @@ msgstr ""
 
 msgid ""
 "Unable to perform boolean operation on model meshes. Only positive parts "
-"will be kept. You may fix the meshes and try again."
+"will be kept. You may fix the meshes and try agian."
```
Comment (Jaizu) [inline localization/i18n/es/OrcaSlicer_es.po:6475]: It's again.

**Review by Jaizu — CHANGES_REQUESTED** [review]
This seems like a messy PR, idk why you changed some English words to misspelled ones.
I made a review pointing the ones that I found and providing some feedback to the spanish strings as well.
This has plenty of changes, so more reviewers are needed, at least someone that confirms that this doesn't mess with original strings and stugg like that.

**Comment by SoftFever** [comment]
@tadeu2 
Thank you. 
Could you have a look at @Jaizu's comments? 
I can't provide an opinion on the Spanish translation, but the English misspellings seem legitimate.

By the way, we can't leave strings empty if translations are not available.

**Comment by tadeu2** [comment]
> This seems like a messy PR, idk why you changed some English words to misspelled ones. I made a review pointing the ones that I found and providing some feedback to the spanish strings as well. This has plenty of changes, so more reviewers are needed, at least someone that confirms that this doesn't mess with original strings and stugg like that.

All the Spanish translation has been done by me from scratch. I have been copying literally, sometimes correcting English mistakes. The first translation I did very quickly and I'm correcting it little by little, seeing spelling mistakes and errors and fixing it. 

If you can help me with it, I'd appreciate it.

File: localization/i18n/es/OrcaSlicer_es.po (hunk 1 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:670]: This is a english original PO mistake

File: localization/i18n/es/OrcaSlicer_es.po (hunk 2 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:917]: Done.

File: localization/i18n/es/OrcaSlicer_es.po (hunk 3 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:964]: There were similar and changed all of them. Thanks.

File: localization/i18n/es/OrcaSlicer_es.po (hunk 4 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:967]: Correct. Changed.

File: localization/i18n/es/OrcaSlicer_es.po (hunk 5 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:1019]: English mistake

File: localization/i18n/es/OrcaSlicer_es.po (hunk 12 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:2940]: I have doubts about these translations. There are a lot of capitalised strings in English, sometimes they seem to be correct and sometimes not. I'm going to correct it as you say anyway

File: localization/i18n/es/OrcaSlicer_es.po (hunk 14 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:4928]: Maybe Softfever should add it. Its a test.

File: localization/i18n/es/OrcaSlicer_es.po (hunk 15 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:6250]: Translated as "Confirmar Guardar como" and also all "Salvar" as "Guardar"

File: localization/i18n/es/OrcaSlicer_es.po (hunk 16 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:6475]: English mistake

File: localization/i18n/es/OrcaSlicer_es.po (hunk 13 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:3714]: English PO mistake

File: localization/i18n/es/OrcaSlicer_es.po (hunk 7 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:2241]: @SoftFever should correct it in original PO.

File: localization/i18n/es/OrcaSlicer_es.po (hunk 8 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:2706]: @SoftFever should correct it in original PO.

File: localization/i18n/es/OrcaSlicer_es.po (hunk 9 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:2715]: Done.

File: localization/i18n/es/OrcaSlicer_es.po (hunk 11 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po]: Fixed.

File: localization/i18n/es/OrcaSlicer_es.po (hunk 10 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po]: Done.

File: localization/i18n/es/OrcaSlicer_es.po (hunk 1 above)
Comment (tadeu2) [reply localization/i18n/es/OrcaSlicer_es.po:670]: @SoftFever should correct it.

**Review by tadeu2 — COMMENTED** [review]
I have corrected the issues. @SoftFever have to review english mistakes.

**Comment by SoftFever** [comment]
> @tadeu2 Thank you. Could you have a look at @Jaizu's comments? I can't provide an opinion on the Spanish translation, but the English misspellings seem legitimate.
> 
> By the way, we can't leave strings empty if translations are not available.

Just realised there's a typo...
 It's supposed to be: "By the way, we can leave strings empty if translations are not available."

**Review by SoftFever — APPROVED** [review]
@tadeu2 @Jaizu Thank you both.

**Comment by tadeu2** [comment]
I dont touched the english po. It can be merged.

**Comment by tadeu2** [comment]
It can be merged mixing the english fixed typos with the translations

**Pull request merged**